config/routes.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'reservations/index'
+  get 'reservations/show'
+  get 'reservations/create'
+  get 'vehicles/index'
+  get 'vehicles/show'
+  get 'vehicles/create'
+  get 'vehicles/destroy'
+  get 'users/index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
@wandergithub, good point. Yes, I know, but they are temporary, since they were generated automatically when I generated the controllers one by one. Thanks  !

Reply:
@wandergithub, regarding whether or not to use api/[version] structure, I know that it's done to be able to change API versions in the future, without affecting old front-end developments that were done before. On the other hand, I have been seeing that the use of api folder  are more used when there are also views to display. Anyway, we could coordinate to change it 👍